index.js

```diff
@@ -76,14 +100,21 @@ var browserslist = function (selections, opts) {
         } else if ( opts.config || process.env.BROWSERSLIST_CONFIG ) {
             var file = opts.config || process.env.BROWSERSLIST_CONFIG;
             if ( fs.existsSync(file) && fs.statSync(file).isFile() ) {
-                selections = browserslist.parseConfig( fs.readFileSync(file) );
+                var conf = browserslist.parseConfig( fs.readFileSync(file) );
+                var envConf = configForEnv(conf, env);
+                if ( envConf !== false ) {
+                    selections = envConf;
+                } else {
+                    selections = browserslist.defaults;
```

Comment:
You get `browserslist.defaults` twice: here and in `configForEnv`. Seems like one of them is unnecessary.